CMakeLists.txt

```diff
@@ -50,9 +36,15 @@ link_directories(${CMAKE_LIBRARY_PATH})
 # Add our library
 add_library(${PROJECT_NAME}_static STATIC ${SOURCES} ${HEADERS})
```

Comment:
If you can, add an alias with ${PROJECT_NAME}-static. Just... Don't ask why.

Reply:
I will ask why. Why?

Reply:
https://github.com/SuperTux/supertux/blob/9bc0425732ac9c4fd9f4e928c16985fbeb066251/mk/cmake/Dependencies.cmake#L57

Reply:
Ah. Well, I pushed the alias